### ISSUE TITLE
Themed/Monochrome App Icon

### ISSUE THREAD

**Issue by NielSync** [aload0/AppLock#1]
Implementing an adaptive/themed app icon would complement the MD3!

![Screenshot_20250606_112800_One UI Home.png](https://github.com/user-attachments/assets/ddcc0ec0-4d9f-4a15-9a2e-bac5fa7650bd)

**Comment by NRG-600** [comment]
I'll be a nice addition and beautiful

**Comment by CompeyDev** [comment]
I second this — it should be a relatively simple addition while improving consistency and even accessibility since monochrome icons are used in several system UI places (such as the notification menu icons, etc).
